src/components/cc-pricing-page/cc-pricing-page.js

```diff
@@ -1,146 +1,189 @@
 import { css, html, LitElement } from 'lit';
```

Comment:
This comment is related to the commit message:

Could you explain which behavior is updated and why smart is removed?

Reply:
Tried my best, feel free to tell me if anything is unclear !